cyberdrop_dl/scraper/crawlers/coomer_crawler.py

```diff
@@ -39,11 +40,33 @@ async def fetch(self, scrape_item: ScrapeItem) -> None:
             await self.post(scrape_item)
         elif "onlyfans" in scrape_item.url.parts or "fansly" in scrape_item.url.parts:
             await self.profile(scrape_item)
+        elif "favorites" in scrape_item.url.parts:
+            if self.manager.config_manager.authentication_data['Coomer']['session']:
+                await self.favorites(scrape_item)
+            else:
+                raise ScrapeFailure("No session cookie found in the config file, cannot scrape favorites.")
```

Comment:
```suggestion
                raise ScrapeFailure(401, message = "No session cookie found in the config file, cannot scrape favorites", origin = scrape_item)
```

Raise as `Unauthorized` (for the UI) and a custom message (for the logs). Also, you can not raise from within `fetch` cause it does not have the `@error_handling_wrapper`. Send `scrape_item` to `self.favorites` and do the cookie check inside it